src/index.css

```diff
@@ -1,13 +1,197 @@
+* {
+  font-family: sans-serif;
+  color: #ffffff;
+}
+
+.main-title {
+  font-family: serif;
+  font-style: italic;
+  font-weight: bold;
+  font-size: 70px;
+  top: 10%;
+  display: flex;
+  justify-content: center;
+}
+
+@keyframes fadein {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+
+.question {
+  font-family: 'MonteCarlo', cursive;
+  animation: fadein 2s;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  display: flex;
```

Comment:
Do you have all of your CSS in a single file? Im so curious! I can see how it would work.

Reply:
Yes and surprisingly it's not the worst lol could use some cleaning up though